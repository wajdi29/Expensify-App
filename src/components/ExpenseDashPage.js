import React from "react";
import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";
const ExpenseDashPage = () => (
  <div>
    <ExpenseListFilters />
    <ExpenseList />
  </div>
);
export default ExpenseDashPage;
