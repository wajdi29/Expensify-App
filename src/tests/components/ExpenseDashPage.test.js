import React from 'react'
import { shallow } from 'enzyme'
import ExpenseDashPage from '../../components/ExpenseDashPage'

test('should render dash page correctly', () => {
    const wrapper = shallow(<ExpenseDashPage />)
    expect(wrapper).toMatchSnapshot()
})