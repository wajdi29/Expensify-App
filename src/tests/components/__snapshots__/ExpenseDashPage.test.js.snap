// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dash page correctly 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
